dashboard/src/components/AppView/AccessURLTable.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from "react";
```

Comment:
I'd like to start actually adding these components inside their own directory, directory that will contain its own index.js as well.

So for example `AppView/AccessURLTable.tsx` will go inside `AppView/AccessURLTable/AccessURLTable.tsx`

I'd like to make it standard across all the components but for now I'll be happy if we do it at least for new components.